input/pagecontent/StructureDefinition-ViewDefinition-notes.md

```diff
@@ -231,6 +231,44 @@ for the resource.)
 
 The [example view definitions](StructureDefinition-ViewDefinition-examples.html) illustrate this behavior.
 
+### unionAll expressions
+The `unionAll` structure can be viewed as a special kind of `select` that effectively concatenates the results of the nested selection structures that it
+contains, but without a guarantee that row ordering will be preserved. Each selection structure contained in the `unionAll` must produce the same column names and FHIR types.
+
+For instance, suppose we want to create a table of all patient and contact addresses. Here's a snippet of what the `unionAll` could look like for that:
+
+```js
+"unionAll": [
+{
+  "select": [
+  {
+    "forEach": "address",
+    "column": [
+      {"path": "postalCode", "name": "zip"},
+      {"path": "true", "name": "is_patient"}
+    ]
+  },
+  {
+    "forEach": "contact.address",
+    "column": [
+      {"path": "postalCode", "name": "zip"}
+      {"path": "false", "name": "is_patient"}
+    ]
+  }]
+}]
+```
+
+The above example uses `forEach` to select parts of the resource for each select to be unioned, but other use cases may get the needed columns directly
```

Comment:
I think the example above has a bug. Wouldn't it be taking a Cartesian product between patient addresses and contact addresses, because these selectors are siblings under `select`?  Like I think you need to remove lines 242 + 243 (and their corresponding closing punctuation marks), no?

Reply:
I lost what this was connected to since this PR has changed a bit since this, but let me know if this comment is still applicable.